broker/xmatch.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3.7
+# -*- coding: UTF-8 -*-
+
+"""This module provides cross matching services between observed targets and
+
+Helpful links:
+  https://astroquery.readthedocs.io/en/latest/#using-astroquery
+  https://astroquery.readthedocs.io/en/latest/xmatch/xmatch.html#module-astroquery.xmatch
+
+Example Usage:
+  from mock_stream import xmatch as xm
+
+  # Write a CSV file with RA, DEC:
+  fradec = 'mock_stream/data/alerts_radec.csv'
+  xm.get_alerts_RA_DEC(fout=fradec)
+
+  Query VizieR for cross matches:
+  table = xm.get_xmatches(fcat1=fradec, cat2='vizier:II/246/out')
+"""
+
+import pandas as pd
+from astropy import units as u
+from astroquery.xmatch import XMatch
+from .mock_ztf_stream import _parse_data as psd
+
+
+def get_xmatches(
+        fcat1='mock_stream/data/alerts_radec.csv', cat2='vizier:II/246/out'):
+    """
+
+    Args:
+        fcat1 (string): Path to csv file, as written by get_alerts_ra_dec.
+        cat2  (string): Passed through to XMatch.query().
+
+    Returns:
+        An astropy table with columns:
+            'angDist','alert_id','ra','dec','2MASS','RAJ2000','DEJ2000',
+            'errHalfMaj','errHalfMin','errPosAng','Jmag','Hmag','Kmag',
+            'e_Jmag','e_Hmag','e_Kmag','Qfl','Rfl','X','MeasureJD'
+    """
+
+    table = XMatch.query(
+        cat1=open(fcat1),
```

Comment:
I'm not sure if the garbage collector will close this file after the function completes. It might be necessary to wrap this with a `with` statement.